proposed/extended-coding-style-guide.md

```diff
@@ -247,6 +247,89 @@ class ClassName extends ParentClass implements
 }
 ```
 
+### Using traits
+
+The `use` keyword used inside the classes to implement traits MUST be
+declared one line after the classe's opening brace, and MUST be indented
+with 4 space.
```

Comment:
The indentation note is unnecessary. Look at how we say similar things for classes (`Note indentation from the example`)